ignite/pkg/sentry/sentry.go

```diff
@@ -0,0 +1,47 @@
+package sentry
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+
+	"github.com/ignite/cli/v29/ignite/version"
+)
+
+const IgniteDNS = "https://bugs.ignite.com"
+
+func InitSentry(ctx context.Context) (deferMe func(), err error) {
+	sentrySyncTransport := sentry.NewHTTPSyncTransport()
+	sentrySyncTransport.Timeout = time.Second * 3
+
+	igniteInfo, err := version.GetInfo(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("failed to init sentry: %w", err)
+	}
+
+	if err := sentry.Init(sentry.ClientOptions{
+		Dsn:         IgniteDNS,
+		Transport:   sentrySyncTransport,
+		Environment: getEnvironment(igniteInfo.CLIVersion),
+		Release:     fmt.Sprintf("ignite@%s", igniteInfo.CLIVersion),
+		SampleRate:  1.0, // get all events
+	}); err != nil {
+		return nil, fmt.Errorf("failed to init sentry: %w", err)
```

Comment:
we need to always use the default ignite errors pkg

Reply:
Yes, I will update. This saved us for sentry btw :D